web/src/components/tasks/business-formation/review/ReviewSignatures.tsx

```diff
@@ -120,7 +121,7 @@ export const ReviewSignatures = (): ReactElement => {
   };
 
   const signers = (): ReactElement => {
-    if (!isCorp && hasNoSigners) {
+    if (!isCorp && !isNonprofit && hasNoSigners) {
```

Comment:
[pebble/sand] Just imo, this would be easier to read if reversed logic. The if (not x and not y and not z) pattern is pretty mind bending.

Obviously our tests should confirm this but I *think* it'd be:

```
if (isCorp || isNonprofit || hasSigners) {
  return displaySigners();
} else {
  return displayEmptySigners();
}
```

Reply:
btw, if you do this refactor, I'd argue that the good ol campground rule suggests you should improve the `incorporators` method below as well

Reply:
done

Reply:
going to do as a separate refactor